doc/introduction/inspecting_circuits.rst

```diff
@@ -85,17 +85,20 @@ For example:
 
 .. code-block:: python
 
-    dev = qml.device('lightning.qubit', wires=(0,1,2,3))
+    dev = qml.device('default.qubit')
```

Comment:
With lightning qubit, the circuit (containing MCMs) below is expanded before drawing, which is not what we want here.

Reply:
Should we make a note of that somewhere for users?

Reply:
I think it's fine to have users discover this themselves, no? Drawing does not strike me as a large-scale "production code" feature. But we maybe should talk _somewhere_ about drawing and (device-controlled) tape expansion :think